Assets/HoloToolkit/Input/Scripts/Voice/SpeechInputSource.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using UnityEngine;
 
-#if UNITY_WSA || UNITY_STANDALONE_WIN
+#if UNITY_WSA || UNITY_STANDALONE_WIN || UNITY_EDITOR
```

Comment:
I take it this is for people that aren't developing on a windows machine?

Reply:
@StephenHodgson `Assets/HoloToolkit/Input/Scripts/Voice/SpeechInputSource.cs(43,16): error CS0246: The type or namespace name 'ConfidenceLevel' could not be found. Are you missing 'UnityEngine.Windows.Speech' using directive?
`

Reply:
Well nevermind this doesn't really solve the issue as it will still fail when building.

Reply:
Oh I see, it fails to build.

Reply:
Ok I don't really know how to solve this then. How can we expose that field, keep the data between platforms and also make it build for all platforms?

Reply:
Hmm... seems to build fine for me.

Reply:
Editor errors with anything not WSA/Standalone_Win
Build errors with my PR

Reply:
Line 43 needs to be moved down to 47

Reply:
```

        [Tooltip("The confidence level for the keyword recognizer.")]
        public ConfidenceLevel RecognitionConfidenceLevel;
```

That's what needs to be moved down into the compile directive

Reply:
Then we'll get the same issues as in #926

Reply:
No getting around this one.

Reply:
make it private and serialize it.

Reply:
Any reason to use private serialization with getter&setter instead? It will have the same effect as just public.

Reply:
```
#if UNITY_WSA || UNITY_STANDALONE_WIN
        [SerializeField]
        [Tooltip("The confidence level for the keyword recognizer.")]
        private ConfidenceLevel recognitionConfidenceLevel;

        public void SetConfidenceLevel(ConfidenceLevel newLevel)
        {
            recognitionConfidenceLevel = newLevel;
        }

        public ConfidenceLevel GetConfidenceLevel()
        {
            return recognitionConfidenceLevel;
        }
```

Reply:
and be sure to update the editor script serialized property lookup too.